decidim-proposals/lib/decidim/proposals/feature.rb

```diff
@@ -31,6 +31,10 @@
           author: Decidim::User.where(organization: feature.organization).all.sample
         )
       end
+
+      Decidim::Proposals::Proposal.all.each do |proposal|
```

Comment:
save line 27 in a variable and use that, so that you don't iterate through all proposals again